.github/workflows/changelog.yml

```
@@ -3,34 +3,48 @@ name: Generate Changelog
         run: |
           git config user.name "github-actions"
           git config user.email "github-actions@github.com"
+          git checkout -b changelog/update-${{ github.run_number }}
           git add CHANGELOG.md
-          git commit -m "Update CHANGELOG.md"
-          git push origin HEAD:master
+          git commit -m "chore: update CHANGELOG.md for ${{ github.ref_name }}"
+          git push origin changelog/update-${{ github.run_number }}
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)